docs/examples/compute/instance_chef/cookbooks/example_webserver/recipes/default.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Cookbook:: example_webserver
```

Comment:
When I run this plan and go to the public IP in my browser, I get an Apache setup page. We could add some content of our own with something like the following (which I've tested, it works):

file '/var/www/html/index.html' do
  content '<html>Hello World!</html>'
end